aineko/extras/connectors/rest_poller.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Extra module for connecting to a REST endpoint."""
+
+import json
+import time
+from typing import Any, Dict, Optional
+
+import requests
```

Comment:
Good question. Correct me I'm wrong but in the particular case of this node, blocking seems ok to me since we are "waiting" for data to come back to the api in order to put them into the dataset. i think this is ok since we are polling on an interval as opposed to trying to fetch data as fast as possible.

Reply:
I agree with Ricardo's reasoning. Sync requests should be fine in this case.

Reply:
That makes sense -- thank you. 

I think that's something we could revisit in the future if we settle on a standard recommended client library later.